pkg/pidfile/pidfile.go

```diff
@@ -0,0 +1,50 @@
+package pidfile
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+)
+
+type PidFile struct {
+	path string
+}
+
+func checkPidFileAlreadyExists(path string) error {
+	if pidString, err := ioutil.ReadFile(path); err == nil {
+		if pid, err := strconv.Atoi(string(pidString)); err == nil {
+			if _, err := os.FindProcess(pid); err == nil {
```

Comment:
A-ha. I just discovered that `FindProcess` *always* returns a `nil` error; that explains the test works, even though the code does not.